spec/amp-versioning-policy.md

```diff
@@ -49,11 +49,9 @@ The following are **not** considered breaking changes:
 
 In rare cases, the AMP project may decide that an existing feature or API must be removed. Such breaking changes must follow the AMP deprecations process prior to removal. Deprecations must be publicly discussed and provide significant user benefit that justifies additional work for page developers.
 
-Process:
-- Create [INTENT TO IMPLEMENT](https://github.com/ampproject/amphtml/labels/INTENT%20TO%20IMPLEMENT) issue.
-- Allow for at least 2 weeks of open discussion.
-- Wait for 3 approvals from core committers.
-- [Announce](#announcements) change on the mailing list.
+The process for deprecations follows the [process for significant changes](https://github.com/ampproject/amphtml/blob/master/contributing/contributing-code.md#process-for-significant-changes) (including the use of Intent-to-implement and Intent-to-ship issues and need for approval from 3 members of the [Approvers Working Group](https://github.com/ampproject/wg-approvers)).  In addition:
+- After the I2I is created, at least 2 weeks of open discussion should be allowed before proceeding with implementation.
```

Comment:
May be unnecessarily restrictive to block implementation on two weeks of discussion. Maybe say that such work should be experimental/prototypal only, and its current implementation should not be a factor in design decisions.

Reply:
I carried the "2 weeks" over from the existing process, but added the wording about implementation to attempt to clarify what waiting 2 weeks meant.  Perhaps that's too strict?

At the extreme case, would we be okay with someone filing an I2I to deprecate a feature, get approvals from 3 Approvers and merging in the deprecation code blocked by an experiment in 1 day, and then waiting the 6 weeks before actually turning the experiment on?

I don't feel strongly either way.  Extra time before implementation means it's easier to incorporate feedback or decide not to go forward with deprecation, but if the code is behind an experiment it's also possible to modify/revert it.

Reply:
I think key is that we need a discussion period. Chrome is using a separate "Intent to Deprecate" which makes sense for us as well. Implementation should typically be trivial.

Reply:
I don't feel strongly either way, either. Just wanted to make sure the change was intended. I suspect in many cases, eager developers will start committing changes to their fork before the 2 weeks, regardless of what this text says. It sounds like that's not what's meant by "implementation" here, but rather landing it in the canonical amphtml repo.

Reply:
For the 2 week period, perhaps change "before starting implementation" to "before the change is merged"? 

@cramforce should I create a new template for the Intent to Deprecate?  How often do we deprecate things?

Reply:
Not very often but I have seen two so far this year and I know of at least one more that is coming.

Reply:
Even though it may be rare it may be good to have it being explicitly different since it (typically) makes documents go from valid to invalid.

Reply:
+1. It is different enough to get its own template.

Reply:
It looks like Chromium also has an "Intent to Remove" in addition to "Intent to Deprecate" according to their [feature removal documentation](https://www.chromium.org/blink/removing-features).  How they interact is a little bit confusing since

- the feature removal documentation suggests you use an Intent to Remove when you know when a feature will be removed and Intent to Deprecate as a discouraged option when you don't know when it will be removed Exactly how it interacts with "Intent to Deprecate" is a bit confusing 
- the [actual templates](https://docs.google.com/document/d/1Z7bbuD5ZMzvvLcUs9kAgYzd65ld80d5-p4Cl2a04bV0/edit) indicates that an Intent to Remove should refer to an Intent to Deprecate, and if you want to remove something immediately it should be "Intent to Deprecate and Remove"

In any case, what should we do for AMP?  "Intent to Deprecate" as the I2I parallel and then just use a normal I2S for it?  Or have a separate "Intent to Remove" as the I2S parallel to ship an Intent to Deprecate?

Reply:
I think separating them is useful if we really only want to deprecate at this time, but we know that usage is too high to be able to remove it any time soon.

TBH we should just follow the Chromium lead here.

Reply:
OK, SG; I'll create new Intent to Deprecate and Intent to Remove templates.

Reply:
New I2R and I2D templates tracked in #22283; once we have those in I'll update this PR.

Reply:
The templates are in, and I've updated this doc to match the new process.  PTAL.